app/src/main/res/layout/vlc_player_interface.xml

```diff
@@ -74,32 +74,54 @@
             android:orientation="horizontal" />
     </FrameLayout>
 
-    <RelativeLayout
+    <androidx.constraintlayout.widget.ConstraintLayout
         android:id="@+id/topPanel"
         android:layout_width="fill_parent"
         android:layout_height="155dp"
+        android:layout_marginTop="20dp"
+        android:layout_marginHorizontal="40dp"
         android:layout_gravity="center_horizontal|top"
         android:clickable="false"
         android:visibility="invisible">
 
+        <LinearLayout
+            android:id="@+id/item_info"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:orientation="vertical"
+            app:layout_constraintEnd_toStartOf="@id/textClock"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent">
```

Comment:
This linearlayout can be removed now with constraints added to the child views (all use a constraint of "start_toendof" the previos child)